src/app/components/popup/popup.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Inject } from '@angular/core';
+import { DialogRef, DIALOG_DATA } from 'src/app/service';
+
+@Component({
+  selector: 'popup',
+  templateUrl: './popup.component.html',
+  styleUrls: ['./popup.component.scss']
+})
+export class PopUpComponent {
+  title: string;
+  submitBtn: string;
```

Comment:
SubmitBtn also might have default value. 

It would be better to specify it is a text: submitBtn->submitBtnText (or submitText)

Reply:
Fixed